src/main/java/com/wellmeet/WellMeet_Recommendation/embedding/service/EmbeddingService.java

```diff
@@ -1,5 +1,27 @@
 package com.wellmeet.WellMeet_Recommendation.embedding.service;
 
-public interface EmbeddingService {
-    float[] createEmbedding(String text);
-}
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.ai.embedding.EmbeddingResponse;
+import org.springframework.ai.openai.OpenAiEmbeddingModel;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class EmbeddingService {
+
+    private final OpenAiEmbeddingModel embeddingModel;
+
+    public float[] createEmbedding(String text) {
+        if (text.isEmpty()) {
+            float[] result = new float[768];
+            return result;
```

Comment:
한번 정의하고 리턴하는 것이 아닌 바로 리턴하는 건 어떤가요?
```suggestion
            return new float[768];
```

Reply:
앗 ㅋㅋ 
result 배열 값 하나 0 아닌 걸로 바꿔서 cosine 유사도 테스트 해 본 건데 다시 원래대로 돌려놓는 것을 깜빡했네요